docs/specification-ligo.md

```diff
@@ -821,19 +860,46 @@ Parameter (in Michelson):
 
 - For `vote` entrypoint with permit, returns the current suitable counter for constructing permit signature.
 
-## Custom entrypoints
+## Startup mode
 
-BaseDAO allows DAOs to define their own additional entrypoints.
+The contract has a startup mode that can be operated managed and exited at the
```

Comment:
>that can be operated managed and exited

Something feels weird here, may be a comma is missing, or one of operated/managed is redundant.